### DISCUSSION TITLE
How to enforce specific ternary spacing and parentheses?

### DISCUSSION THREAD

**Discussion by Kabbah** [eslint/eslint#14075] [Q&A] [answered]
I personally like to put the condition of a ternary operator around parentheses and without a space before `?`, like so:
```js
const x = (condition)? "value1" : "value2";
```

Is there a way of enforcing this style with ESLint?

**Comment by mdjermanovic** [comment, marked as answer]
Hi @Kabbah!

ESLint doesn't have core rules/options that would enforce this style.

The core stylistic rules [are frozen](https://github.com/eslint/eslint#stylistic-rule-updates), but you can always [create a plugin](https://eslint.org/docs/developer-guide/working-with-plugins).

**Reply by Kabbah** [reply]
Thank you @mdjermanovic, I'm gonna look into it.
